README.md

```diff
@@ -22,7 +22,7 @@ See the 'Running docker on your machine' section at the bottom of this README.
 ### Create Conda environment
 Setup a conda environment to run in using the following command run from the repository folder:
 
-```
+```bash
```

Comment:
``` #!/bin/bash
```bash
```

Reply:
Good to clarify the code type / scripting environment. I think "#!/bin/bash" is the convention to indicate a script is bash. It's often at the top to of a bash script.

Reply:
added #!/usr/bin/env bash